book/src/development/lint_passes.md

```diff
@@ -0,0 +1,137 @@
+# Lint passes
```

Comment:
So this chapter already exists. I think there is a little documentation missing on what a lint pass is in general and how it works together with rustc.